pkg/controllers/workgenerator/controller.go

```diff
@@ -388,11 +464,49 @@ func (r *Reconciler) SetupWithManager(mgr ctrl.Manager) error {
 					return
 				}
 				// Make sure the work is not deleted behind our back
+				klog.V(2).InfoS("Received a work delete event", "work", klog.KObj(evt.Object), "parentBindingName", parentBindingName)
```

Comment:
```suggestion
				klog.V(3).InfoS("Received a work delete event", "work", klog.KObj(evt.Object), "parentBindingName", parentBindingName)
```
changed to v3? otherwise, we'll have too many logs

Reply:
our prod log level is set to v2, so this change will bump it out of the prod log. I am not sure if we will have too many logs of work deleting. Is any reason that you are worried?

Reply:
The project's log level is not debug. We need those logs to debug livesite. If we bump those higher than we can't debug those. We can adjust them after a while when we are more confident in the logic

Reply:
sure, that works too. usually i set the log level high in the code, and then adjust the log flag to print more logs in the production.

anyway, we could revisit the log level once we have the confidence on our code. i notice that a lot of places are using v2.